### DISCUSSION TITLE
Advice for Newbies!!!! Don't install the old nora

### DISCUSSION THREAD

**Discussion by zeroalphagit** [andrei-tatar/node-red-contrib-smartnora#108] [General]
I had all sorts of problems trying to get this to work because I installed the old node-red-contrib-nora first. As soon as I uninstalled the old one and reinstalled this new one everything worked instantly.... 

If you see the old blue colored nodes then you need to uninstall it. This new one has Yellow Nodes

**Comment by andrei-tatar** [comment]
They should be 100% independent. Having both installed shouldn't impact the other. Just make sure you use the new nodes with smart-nora and the old ones with the old heroku instance.
